lib/tasks/integration.rake

```diff
@@ -147,3 +86,104 @@ namespace :integrate do
   end
 
 end
+
+module TestOrchestrationHelpers
```

Comment:
BTW, you can define this as an anonymous module inside the namespace:


```ruby

namespace :integrate do
  helpers = Module.new do
    module_function

    def somehelper
    end
  end

  task :foo do
    helpers.somehelper
  end
end
```

Reply:
TIL ;)

Reply:
actually, need some help on this one @mikz 

```
NoMethodError: undefined method `test_commands' for #<Module:0x00000001f2f4f8>
14:57:51 [7] /opt/system/lib/tasks/integration.rake:111:in `block in <main>'
```

which happens here: 
https://github.com/3scale/porta/blob/more_flexible_tests_execution/lib/tasks/integration.rake#L111

but not sure why... ? 🤔 

(googling isn't proving very helpful...)